packages/wonder-blocks-core/components/view.js

```diff
@@ -40,9 +51,20 @@ const StyledDiv = addStyle<"div">("div", styles.default);
  * - An `aphrodite` StyleSheet style
  * - An array combining the above
  */
-export default class View extends React.Component<TextViewSharedProps> {
+export default class View extends React.Component<Props> {
     render() {
-        const {testId, ...restProps} = this.props;
-        return <StyledDiv data-test-id={testId} {...restProps} />;
+        const {testId, tag, ...restProps} = this.props;
+        switch (tag) {
+            case "article":
+                return <StyledArticle data-test-id={testId} {...restProps} />;
+            case "aside":
+                return <StyledAside data-test-id={testId} {...restProps} />;
+            case "nav":
+                return <StyledNav data-test-id={testId} {...restProps} />;
+            case "section":
+                return <StyledSection data-test-id={testId} {...restProps} />;
+            default:
+                return <StyledDiv data-test-id={testId} {...restProps} />;
```

Comment:
Once `tag` is required can you add a `case "div": ...` and then follow https://medium.com/@ibosz/advance-flow-type-1-exhaustive-checking-with-empty-type-a02e503cd3a0 so that we can use `flow` to ensure this switch statement is exhaustive.  The benefit of this is that if we update `ValidViewTags`, flow will let us know to update the switch/case statement here.